pkg/data/management/role_data.go

```diff
@@ -85,6 +85,8 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 	userRole.addNamespacedRule("cattle-global-data").addRule().apiGroups("").resources("secrets").verbs("create")
 
 	rb.addRole("User Base", "user-base").
+		// standard permissions for regular users, on their tokens
+		addRule().apiGroups("ext.cattle.io").resources("tokens").verbs("get", "list", "watch", "create", "delete", "update", "patch").
```

Comment:
Should we add this to the [user rules](https://github.com/rancher/rancher/blob/main/pkg/data/management/role_data.go#L80) too?

Reply:
Done with commit 73501deeb6e905088f0eab7040de8c26f348d4b9 (rebase!)